test/integration/targets/aws_waf_web_acl/tasks/main.yml

```diff
@@ -0,0 +1,421 @@
+- block:
+  - name: set yaml anchor
```

Comment:
Is this needed + `aws_connection_info` in the default vars?

Reply:
The one in defaults does nothing. So I'll delete that.